pre-registration/pre-registration-application-service/src/main/java/io/mosip/preregistration/application/controller/LoginController.java

```diff
@@ -91,16 +104,21 @@ public void initBinder(WebDataBinder binder) {
 	 * @param errors
 	 * @return AuthNResponse
 	 */
+	@Deprecated(since = "1.1.6", forRemoval = true)
 	@PostMapping(value = "/sendOtp", produces = MediaType.APPLICATION_JSON_VALUE)
-	@Operation(summary  = "Send Otp to UserId")
+	@Operation(summary = "Send Otp to UserId")
 	@ResponseStatus(value = HttpStatus.OK)
 	public ResponseEntity<MainResponseDTO<AuthNResponse>> sendOTP(
 			@Validated @RequestBody MainRequestDTO<OtpRequestDTO> userOtpRequest, @ApiIgnore Errors errors) {
-		log.info("sessionId", "idType", "id", "In sendOtp method of Login controller for sending Otp ");
+		log.info("In sendOtp method of Login controller for sending Otp ");
```

Comment:
Until u need to really inform something to the log there is no reason to print into log. The same logic you have to apply everywhere. We can enable access log and see the API calls. So no need to tell explicitly that you are inside this method. Access log will print the invoked API and the HTTP codes.